test/tools/libtesteth/ImportTest.cpp

```diff
@@ -50,6 +50,25 @@ int mainnetChainID()
         ChainParams(genesisInfo(eth::Network::MainNetworkTest)).chainID;
     return c_mainnetChainID;
 }
+
+ChainParams createChainParamsForNetwork(string const& _networkName)
+{
+    // network name may be network+EIPs
+    vector<string> networkAndEIPs;
+    boost::algorithm::split(networkAndEIPs, _networkName, boost::is_any_of("+"));
```

Comment:
There's no appropriate STL function that we can use?

Reply:
Not really, string functions are pretty low-level and/or verbose (like you could of cource `string::find` in a loop)

I like boost string functions because they create higher-level code, e.g. similar to split/join functions of other languages.

(We're probably going to get split via ranges in C++20 https://stackoverflow.com/questions/48402558/how-to-split-a-stdstring-into-a-range-v3-of-stdstring-views)